tokens/blocket.se/semantic.yml

```diff
@@ -3,10 +3,6 @@ token: maps
 duplicate_as_rgb: true
```

Comment:
Since there is no color tokens left in these files, we should be able to remove the rgb duplication flag.
```suggestion
```

Reply:
Nice! Updated in this commit: [8f02aa1](https://github.com/warp-ds/css/pull/219/commits/8f02aa13494170ed451d0a11cf9b47dfd2f34441)